src/server/snap.rs

```diff
@@ -185,6 +188,20 @@ pub fn send_snap(
             Ok(_) => {
                 fail_point!("snapshot_delete_after_send");
                 mgr.delete_snapshot(&key, &chunks.snap, true);
+                let cost = UnixSecs::now().into_inner() - snapshot_tracker.start;
+                // it should ignore if the duration of snapshot is less than 1s to decrease the
```

Comment:
why not do some aggregation in tikv side?

Reply:
PD will decide the count of operator depences on these duration, it can adjust by some configuration.